controllers/pytorch/pytorchjob_controller.go

```diff
@@ -193,7 +193,7 @@ func (r *PytorchJobReconciler) SetClusterSpec(job interface{}, podTemplate *core
 	}
 
 	masterAddr := job_controller.GenGeneralName(pytorchJob.Name, strings.ToLower(string(pytorchv1.PyTorchReplicaTypeMaster)), strconv.Itoa(0))
-	if v1.ReplicaType(rtype) == pytorchv1.PyTorchReplicaTypeMaster {
+	if rtype == strings.ToLower(string(pytorchv1.PyTorchReplicaTypeMaster)) {
```

Comment:
are there other places require this conversion too

Reply:
it seems no such issue in other referenced expressions.